### PR TITLE
hide community row if nsfw content is hidden

### PR DESCRIPTION
**Pull request by seanyesmunt**
I think we should keep the "row" styling somewhat there. Then users know this would be a scrollable row just like the other rows, if they change their settings.

screen shot 2018-07-11 at 9 54 28 pm

### CODE REVIEW

File: src/renderer/scss/component/_card.scss [hunk 1]
```diff
@@ -21,7 +21,6 @@
 }
 
 .card--small {
-  width: var(--card-small-width);
```
Comment (seanyesmunt) [inline src/renderer/scss/component/_card.scss]: this wasn't doing anything.

File: src/renderer/scss/component/_card.scss [hunk 2]
```diff
@@ -283,6 +282,12 @@
   &:last-of-type {
     padding-bottom: $spacing-vertical * 2/3;
   }
+
+  // For the message if community top bids are disabled
+  .card__content.help {
```
Comment (seanyesmunt) [inline src/renderer/scss/component/_card.scss]: This is needed because of the special CSS that happens inside the CategoryList component, that allows content to scroll "off" the side of the edge. That behavior doesn't exist anywhere else in the app.

File: src/renderer/component/categoryList/view.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 // @flow
-import React from 'react';
+import * as React from 'react';
```
Comment (skhameneh) [inline src/renderer/component/categoryList/view.jsx]: why this change?

Comment (seanyesmunt) [reply src/renderer/component/categoryList/view.jsx]: It's needed after adding `React.Fragment`. Flow acts funny without it.

**Comment by tzarebczan** [comment]
This looks good to me and confirmed with Rob that will move trending to top (or 2nd from top) so that Community top bids will be at the bottom. This means we won't have to worry about the weird spacing.
